requirements/base.txt

```diff
@@ -12,7 +12,7 @@ git+https://github.com/edx/edx-submissions.git@1.1.0#egg=edx-submissions==1.1.0
 boto>=2.32.1,<3.0.0
 celery==3.1.18
 defusedxml==0.4.1
-django>=1.8,<1.9
+django==1.8.13
```

Comment:
This is the version that's already installed via edx-platform, specifying it exactly vs a range should resolve the warning.